lib/manager/pluginManager.js

```diff
@@ -3,114 +3,123 @@ const path = require("path");
 const git = require("simple-git")("./sharedPlugins");
 
 
-module.exports = function(config){
-	if(!config) config = require("./../../config");
-	
-	return {
-		listPlugins: async function(){
-			let plugins = await this.getPlugins();
-			plugins.forEach(plug => {
-				console.log(`${plug.name}`.padEnd(22).substr(0,22)+` - ${plug.description}`.substr(0,80-22));
-			});
-		},
-		getPlugins: async function(){
-			let directories = await fs.readdir("./sharedPlugins");
-			let returnArray = [];
-			for(let i in directories){
-				let dir = directories[i];
-				let pluginContents = await fs.readdir(path.join("./sharedPlugins", dir));
-				// console.log(pluginContents)
-				if(pluginContents.includes("config.js") || pluginContents.includes("config.json")){
-					try{
-						var pluginConfig = require(path.join("./../../sharedPlugins", dir, "config"));
-						pluginConfig.pluginPath = path.join("./sharedPlugins", dir);
-					}catch(e){
-						console.log(e)
-					}	
-					
-					// check if plugin is enabled
-					try{
-						await fs.stat(path.join(pluginConfig.pluginPath, "DISABLED"))
-						pluginConfig.enabled = false;
-					}catch(e){
-						pluginConfig.enabled = true;
-					}
-					returnArray.push(pluginConfig);
-				}
+async function listPlugins() {
+	let plugins = await this.getPlugins();
+	plugins.forEach(plug => {
+		console.log(`${plug.name}`.padEnd(22).substr(0,22)+` - ${plug.description}`.substr(0,80-22));
+	});
+}
+
+async function getPlugins() {
+	let directories = await fs.readdir("./sharedPlugins");
+	let returnArray = [];
+	for(let i in directories){
+		let dir = directories[i];
```

Comment:
No.  `i` is the index in the array of directories and `dir` is the given directory.  This would be better written as `for (let dir of directories) {` though.